tw-tasks-core/src/main/java/com/transferwise/tasks/dao/MySqlTaskDao.java

```diff
@@ -154,6 +155,9 @@ public void init() {
     grabForProcessingSql = "update " + taskTable + " set processing_client_id=?,status=?"
         + ",processing_start_time=?,next_event_time=?,processing_tries_count=processing_tries_count+1"
         + ",state_time=?,time_updated=?,version=? where id=? and version=? and status=?";
+    grabForProcessingWithoutStatusAssertionSql = "update " + taskTable + " set processing_client_id=?,status=?"
```

Comment:
Perhaps we can call this `grabForProcessingSql` as it is now a default behaviour.